src/registry.rs

```diff
@@ -234,7 +234,7 @@ impl Registry {
     /// let subsystem_b_counter_1: Counter = Counter::default();
     /// let subsystem_b_counter_2: Counter = Counter::default();
     ///
-    /// let subsystem_a_registry = registry.sub_registry_with_prefix("subsystem_b");
+    /// let subsystem_b_registry = registry.sub_registry_with_prefix("subsystem_b");
```

Comment:
Good catch. Nit pick, would you mind proposing this in a separate pull request?

Reply:
Sure! I can do that